horovod/run/run.py

```diff
@@ -50,18 +50,18 @@
 
 @cache.use_cache()
 def _check_all_hosts_ssh_successful(host_addresses, ssh_port=None):
-    """
+    '''
```

Comment:
Docstrings need to be double-quotes in Python.  See: https://www.python.org/dev/peps/pep-0257/